website/source/docs/provisioners/chef.html.markdown

```diff
@@ -77,6 +77,12 @@ The following arguments are supported:
   subdirectory called `logfiles` created in your current directory. The filename will be
   the `node_name` of the new node.
 
+* `use_policyfile (boolean)` - (Optional) Use Chef Policyfiles.
```

Comment:
Could be expanded out to identify what this setting does if true, and what the default is.